waltid-libraries/waltid-sdjwt/src/jvmMain/kotlin/id/walt/sdjwt/SimpleJWTCryptoProvider.kt

```diff
@@ -33,3 +33,16 @@ class SimpleJWTCryptoProvider(
     }
 }
 
+class SimpleMultiKeyJWTCryptoProvider(
```

Comment:
is it still required? looks like it's not used

Reply:
used it in a test